e2-studio-star-rx72n-firmware/libs/rx_comm_manager/src/rx_comm_manager.c

```
@@ -1487,11 +1511,19 @@ rx_err_t rx_comm_manager_send(rx_comm_manager_t* mgr, const rx_comm_send_params_
         rx_log_error(s_tag, "Send failed: SPI handle NULL");
         return k_rx_err_invalid_state;
       }
-      err = rx_spi_comm_send(mgr->spi_handle,
-                             params->type,
-                             params->flags,
-                             params->payload,
-                             params->payload_len);
+      /* Use HARQ+FEC link layer if available, otherwise raw SPI */
+      if (mgr->spi_link != nullptr) {
```

Clang (14.0.6)

[warning] if with identical then and else branches

(bugprone-branch-clone)